src/haskell/install.sh

```diff
@@ -1,23 +1,29 @@
 #!/usr/bin/env bash
 set -e
 
+GHCUP_VERSION="0.1.19.2"
+GHCUP_BIN="x86_64-linux-ghcup-${GHCUP_VERSION}"
```

Comment:
```suggestion
# Source /etc/os-release to get OS info
. /etc/os-release
# Fetch host/container arch.
architecture="$(dpkg --print-architecture)"
GHCUP_BIN="${architecture}-linux-ghcup-${GHCUP_VERSION}"
```

Reply:
Done, though I just used the `arch` command instead because /etc/os-release returns `amd64` instead of `x86_64`